sdk/storage/inc/blobs/internal/blob_client_options.hpp

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <string>
+#include <limits>
+#include <utility>
+
+#include "protocol/blob_rest_client.hpp"
+
+namespace Azure { namespace Storage { namespace Blobs {
+
+  struct BlobClientOptions
+  {
+    std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> policies;
+  };
+
+  struct BlockBlobClientOptions : public BlobClientOptions
```

Comment:
or where is it supposed to be?

Reply:
If you mean there should be a separate `block_blob_client_options.hpp`, I think I ever mentioned it to you the other day. Because I feel these `xx_options.hpp` files are quite light-weighted. Maybe we can put options of all three kinds of blob into one file.

@katmsft it would be more much helpful if you can question with suggestion, so that we don't need to waste alot of time replying back and forth or guessing what each other means.

Reply:
we talked about this today and came to a conclusion.